tests/som/tests/BasicInterpreterTests.java

```diff
@@ -114,7 +114,7 @@
         {"Regressions", "testSymbolEquality", 1, SInteger.class},
         {"Regressions", "testSymbolReferenceEquality", 1, SInteger.class},
 
```

Comment:
This is missing, I believe. Not tested.
But, you can't just increase the number of tests without adding the test :)

```suggestion

        {"BinaryOperation", "test", 3 + 8, SInteger.class},

```

Reply:
Oops. 
By the way, any reason `{"Self", "assignSuper", 42, ProgramDefinitionError.class}` is commented (it is green)? With this one not executed, we effectively have 51 tests and not 52

Reply:
Yeah, that's fine. If I remember correctly, that one tests whether we can assign to super, and we should really not be able to do that.